### ISSUE TITLE
The notification icon, should change for show unread notifies

### ISSUE THREAD

**Issue by Brummell67** [cairoshell/cairoshell#330]
As title:
![image](https://user-images.githubusercontent.com/5261273/73397670-52f53600-42e4-11ea-9ac5-7ee2e2383216.png)

Something like this:
![image](https://user-images.githubusercontent.com/5261273/73398127-2db4f780-42e5-11ea-91a3-a039a3340cbe.png)

**Comment by dremin** [comment]
Agree! I wonder if there is an API that allows us to get that information.

**Comment by Brummell67** [comment]
Maybe this:
https://docs.microsoft.com/en-us/uwp/api/windows.ui.notifications.management.usernotificationlistener.getnotification

**Comment by josuave** [comment]
@dremin That is interesting. I can’t imagine the explorer is changing that icon on its own. Maybe an undocumented notify message to update the icon? I think this is provided by shell service objects?
